templates/web/angus/report/new/category_extras.html

```diff
@@ -0,0 +1,29 @@
+<div id="category_meta">
```

Comment:
If the only difference here is the lack of `h4` (if I've diffed it right), hide it with CSS instead?